soot-infoflow-android/src/soot/jimple/infoflow/android/source/AndroidSourceSinkManager.java

```diff
@@ -937,10 +973,17 @@ public void initialize() {
 					SootMethod sm = Scene.v().grabMethod(entry.getKey());
 					if (sm != null)
 						sinkMethods.put(sm, entry.getValue());
+					else{//in case method signature doesn't contain return type
+						String className=((MethodSourceSinkDefinition) sourceSinkDef).getMethod().getClassName();
```

Comment:
only cast once and store the object in a variable

Reply:
Done